ci-operator/step-registry/sandboxed-containers-operator/get-kata-rpm/sandboxed-containers-operator-get-kata-rpm-commands.sh

```diff
@@ -22,15 +22,46 @@ fi
 
 brew_auth="$(oc get -n openshift-config secret/pull-secret -ojson  | jq -r '.data.".dockerconfigjson"' |  base64 -d | jq -r '.auths."registry.redhat.io".auth' | base64 -d)"
 
-md5sum_file="${KATA_RPM_BUILD_MD5SUM}  kata-containers.rpm"
-curl -L -k -o kata-containers.rpm -u "${brew_auth}" "${KATA_RPM_BUILD_URL}"
-echo "Debug:"
-ls -l kata-containers.rpm || true
-echo "Checking against md5sum ${KATA_RPM_BUILD_MD5SUM}"
-echo "${md5sum_file}" | md5sum -c -
+# md5sum_file="${KATA_RPM_BUILD_MD5SUM}  kata-containers.rpm"
+OUTPUT="$(curl -L -k -o kata-containers.rpm -u "${brew_auth}" "${KATA_RPM_BUILD_URL}" 2>&1)"
+err=$?
+if [ $err -ne 0 ]; then
+    echo "ERROR: curl error ${err} trying to get ${KATA_RPM_BUILD_URL}"
+    echo "ERROR: ${OUTPUT}"
+    exit $err
```

Comment:
It might be better print the `err` but return hardcoded (`1`) return code to simplify finding where this script failed